restapi.go

```diff
@@ -2755,3 +2755,127 @@ func (s *Session) FollowupMessageEdit(appID string, interaction *Interaction, me
 func (s *Session) FollowupMessageDelete(appID string, interaction *Interaction, messageID string) error {
 	return s.WebhookMessageDelete(appID, interaction.Token, messageID)
 }
+
+// ------------------------------------------------------------------------------------------------
+// Functions specific to guilds scheduled events
+// ------------------------------------------------------------------------------------------------
+
+// GuildScheduledEvents returns an array of GuildScheduledEvent for a guild
+// guildID        : The ID of a Guild
+// userCount      : Whether to include the user count in the response
+func (s *Session) GuildScheduledEvents(guildID string, userCount bool) (st []*GuildScheduledEvent, err error) {
+	uri := EndpointGuildScheduledEvents(guildID)
+
+	queryParams := url.Values{}
+	if userCount {
+		queryParams.Set("with_user_count", "true")
+	}
```

Comment:
Not sure if `url.Values` is necessary here. `with_user_count` could just be added directly to the uri.

Reply:
I personally prefer to structure and leave the possibility to add more query params after without rewrite the function. Tell me if you prefer append string only, If that's the case, I'll make the change without worrying :)

Reply:
That's just the convention. We use `url.Values` only when there's multiple query params.